array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,42 @@
+/* exported takeRight */
+
+/*
+  - Create new array variable
+  - Loop through array, with initialization at array.length - count
+    - If count is greater than array length,
+      - then return array input
+    - Else push array at current index into new array
+  - Return new array
+*/
+
+function takeRight(array, count) {
+  var newArray = [];
+  for (var i = array.length - count; i < array.length; i++) {
+    if (count > array.length) {
+      return array;
```

Comment:
It's best to get guards like this out of the way before your loop even starts, since you only need to check the condition on line 15 once.